std/experimental/allocator/mallocator.d

```diff
@@ -198,7 +212,8 @@ struct AlignedMallocator
     /**
     Forwards to $(D alignedAllocate(bytes, platformAlignment)).
     */
-    @nogc @trusted void[] allocate(size_t bytes) shared
+    @trusted @nogc nothrow
```

Comment:
When I was writing this, I knew that your [MMM PR](https://github.com/D-Programming-Language/phobos/pull/3031) was rejected (I didn't remember why), so I decided to leave this attribute as it is, in order to make it non-controversial for merging.

I now see that you have a new [PR](https://github.com/D-Programming-Language/phobos/pull/3891) that enhances attribute inference of the `typed` interface of `stdx.allocator`, so if that goes in, I'll be fine with closing by request.

I just hope that we can merge this into stable, so we can have these fixes in time for DMD 2.070.